components/Modal/DeleteBlueprint.js

```diff
@@ -30,9 +30,11 @@ class DeleteBlueprint extends React.Component {
                 className="close"
                 data-dismiss="modal"
               >
-                <span className="pficon pficon-close"></span>
+                <span 
+                className="pficon pficon-close"
+                id="pficon-close"></span>
               </button>
-              <h4 className="modal-title" id="myModalLabel"><FormattedMessage defaultMessage="Delete Blueprint" /></h4>
+              <h4 className="modal-title" data-id="myModalLabel"><FormattedMessage defaultMessage="Delete Blueprint" /></h4>
```

Comment:
In this case we want the `id`. 

The rules to follow for `data-id` vs `id` are:
- If we need to select an element, check if an `id` attribute already exists.
  - If `id` already exists, then use the `id`
  - If `id` doesn't exist, then check if the element is using a class that no other element in that control is using
    - If a fairly unique `class` exists, then use the class
    - If a class doesn't exist or is used by other elements, then add a `data-id` attribute with a value that identifies the element.

If an `id` is used, then it's being used to associate html elements with other elements. This is most likely going to be used in form elements. But here, the `id` value is being referenced by the modal parent `<div>` in the attribute `aria-labelledby="myModalLabel"` (i.e. the label for this modal is provided by this `<h4>` element)